content/en/docs/collector/getting-started.md

```diff
@@ -171,12 +171,14 @@ locally to a file. Data is sent to the container and the container scrapes its o
 Prometheus metrics.
 
 ```console
-$ git clone git@github.com:open-telemetry/opentelemetry-collector-contrib.git; \
-    cd opentelemetry-collector-contrib/examples/demo; \
-    go build client/main.go; ./client/main & pid1="$!"; \
-    go build server/main.go; ./server/main & pid2="$!"; \
-
-$ git clone git@github.com:open-telemetry/opentelemetry-collector.git; \
+$ git clone https://github.com/open-telemetry/opentelemetry-collector-contrib.git; \
+    cd opentelemetry-collector-contrib/examples/demo/client; \
+    go build -o main main.go; ./main & pid1="$!"; \
+    cd ../server; \
+    go build -o main main.go; ./main & pid2="$!"; \
+    cd ../../../..
+
+$ git clone https://github.com/open-telemetry/opentelemetry-collector.git \
```

Comment:
I'd suggest using `&&` instead of `;` where it's not possible to continue without the operation succeeding

```suggestion
$ git clone https://github.com/open-telemetry/opentelemetry-collector.git; \
```

Reply:
I have added the suggested change. Thanks!